scania-digital-design-system/tegel
# test(popover-canvas): Adding unit tests

**Describe pull-request**  
adding unit tests for popover canvas.

**Solving issue**  
Fixes: [CDEP-3007](https://tegel.atlassian.net/browse/CDEP-3007)

**How to test**  
1. Checkout code
2. Run `npm run test` and see that all passes


[CDEP-3007]: https://tegel.atlassian.net/browse/CDEP-3007?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
